File: problems/hacker_rank/month/Day2.java
package hacker_rank.month;

import java.util.Scanner;

class Day2 {
    public static void main(String[] arg) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        Scanner scan = new Scanner(System.in);

        int v = scan.nextInt();
        scan.nextLine();
        double de = scan.nextDouble();
        scan.nextLine();
        String string = scan.nextLine();

        double r = d + de;

        r = Math.round(r * 10.0) / 10.0;

        System.out.println(i+v);
        System.out.println(r);
        System.out.println(s+string);

    }

}

File: problems/hacker_rank/month/Day2.kt
package hacker_rank.month

fun main(args: Array<String>) {
    val i = 4
    val d = 4.0
    val s = "HackerRank "

    val int = readLine()?.toInt() ?: 0
    val double = readLine()?.toDouble() ?: 0.0
    val string = readLine() ?: ""

    var r= d+double

    r = Math.round(r * 10.0) / 10.0

    println(i + int)
    println(r)
    println(s + string)
}